src/utils/api/metacom.ts

```diff
@@ -1,7 +1,7 @@
 'use client'
 
-import { useToast } from '@/components/ui/use-toast'
 import { URL_PRODUCT_API } from '@/constants'
+import { useToast } from '@/utils/api/hooks/use-toast'
```

Comment:
I think it should be `import { useToast } from '@/utils`

Reply:
> I think it should be `import { useToast } from '@/utils`

The problem is that you've placed custom hook somewhere in the nested folders but not in the hooks folder itself. On my opinion, it would be better to just create src/hooks directory and place all our custom hooks in there - this is the common practice for custom hooks

Reply:
Let it be, I'd create some `src/hooks` and place some hooks there